test/snapshots/index.test.js.md

```diff
@@ -2186,8 +2185,8 @@ Generated by [AVA](https://avajs.dev).
         generate: 'nuxt generate',
         start: 'nuxt start',
         test: 'ava',
-        'test:e2e': 'cross-env TEST=e2e ava --config e2e.config.js',
-        'test:unit': 'cross-env TEST=unit ava --config unit.config.js',
+        'test:e2e': 'cross-env TEST=e2e ava --config ava.config.cjs ./test/e2e/**/*',
```

Comment:
Can `--config ava.config.cjs` be ignored ?

Reply:
Yep. As it'll automatically find a ava.config.* file in the root dir.

I'll update this.